### PR TITLE
Docsite: misc fixes

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Docsite: misc fixes of best practices

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/dev_guide/developing_modules_best_practices.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @felixfontein

File: docs/docsite/rst/dev_guide/developing_modules_best_practices.rst [hunk 1]
```diff
@@ -135,7 +135,7 @@ Modules must output valid JSON only. Follow these guidelines for creating correc
 * Make returns reusable--most of the time you don't want to read it, but you do want to process it and re-purpose it.
 * Return diff if in diff mode. This is not required for all modules, as it won't make sense for certain ones, but please include it when applicable.
 * Enable your return values to be serialized as JSON with Python's standard `JSON encoder and decoder <https://docs.python.org/3/library/json.html>`_ library. Basic python types (strings, int, dicts, lists, and so on) are serializable.
-* Do not return an object via exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
+* Do not return an object trough exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
```
Comment (acozine) [inline docs/docsite/rst/dev_guide/developing_modules_best_practices.rst]: +1 for getting rid of "via". In this instance I'd use "using" or "with" instead:
```suggestion
* Do not return an object using exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
```

File: docs/docsite/rst/dev_guide/developing_modules_best_practices.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ General guidelines & tips
 * Each module should be self-contained in one file, so it can be auto-transferred by ``ansible-base``.
 * Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent ``ansible-base`` from importing your module.
 * Always use the ``hacking/test-module.py`` script when developing modules - it will warn you about common pitfalls.
-* If you have a local module that returns facts specific to your installations, a good name for this module is ``site_facts``.
+* If you have a local module that returns facts specific to your installations, a good name for this module is ``site_info``.
```
Comment (acozine) [inline docs/docsite/rst/dev_guide/developing_modules_best_practices.rst]: We should use "info" or "facts" consistently:
```suggestion
* If you have a local module that returns information specific to your installations, a good name for this module is ``site_info``.
```

**Review by acozine — COMMENTED** [review]
Looks good, thanks @Andersson007. I see a couple of places where it could be even better.

File: docs/docsite/rst/dev_guide/developing_modules_best_practices.rst (hunk 1 above)
Comment (Andersson007) [reply docs/docsite/rst/dev_guide/developing_modules_best_practices.rst]: i also had feeling that there should've been another word:)

File: docs/docsite/rst/dev_guide/developing_modules_best_practices.rst (hunk 2 above)
Comment (Andersson007) [reply docs/docsite/rst/dev_guide/developing_modules_best_practices.rst]: good spot! thanks

**Comment by Andersson007** [comment]
ready_for_review

**Review by samccann — APPROVED** [review]

**Pull request merged**